ast/desugar/Desugar.cc

```diff
@@ -240,8 +240,8 @@ ExpressionPtr desugarDString(DesugarContext dctx, core::LocOffsets loc, parser::
         return mergeStrings(dctx, loc, std::move(stringsAccumulated));
     } else {
         auto recv = MK::Constant(loc, core::Symbols::Magic());
-        return MK::Send(loc, std::move(recv), core::Names::stringInterpolate(), interpArgs.size(),
-                        std::move(interpArgs));
+        return MK::Send(loc, std::move(recv), core::Names::stringInterpolate(), loc.copyWithZeroLength(),
```

Comment:
So just fwiw I started to read through all of these to make sure the choice of loc made sense to me, and got about a quarter of the way through the file not having seen anything objectionable, and then realized how long it was and decided that was enough scrutiny. If there's less mechanical stuff in this file I missed maybe let me know :)

Reply:
yeah it's all mechanical.